src/isPalindrome.js

```diff
@@ -19,7 +19,10 @@
  * @return {boolean}
  */
 function isPalindrome(str) {
-  // write code here
+  const word = str.replace(/[\W_]/g, '').toLowerCase();
```

Comment:
word - слово, у тебя может быть и целое предложение
```suggestion
  const replacedString = str.replace(/[\W_]/g, '').toLowerCase();
```